backend-transit/src/main/java/eu/socialedge/hermes/backend/transit/domain/Route.java

```diff
@@ -64,7 +64,7 @@ public Route(String code, VehicleType vehicleType, List<Station> stations, Shape
         this.vehicleType = notNull(vehicleType);
         this.stations = new ArrayList<>(notEmpty(stations));
 
-        if (!containsAllStations(shape))
+        if (!containsAllStations(notNull(shape)))
```

Comment:
If it's a generator's requirement, let it generates Shape by itself. Route without shape is perfectly fine without a shape.